src/connecpy/async_client.py

```diff
@@ -112,10 +137,10 @@ async def _make_request(
                 return response
             else:
                 raise ConnectWireError.from_response(resp).to_exception()
-        except httpx.TimeoutException as e:
+        except (httpx.TimeoutException, TimeoutError):
             raise exceptions.ConnecpyServerException(
                 code=errors.Errors.DeadlineExceeded,
-                message=str(e) or "request timeout",
```

Comment:
For timeout errors, I think the error message is almost always an implementation detail so just changed it to the fixed string